### PR TITLE
Resolving Revolting Issue

### PR DESCRIPTION
**Pull request by arihant-001**
with reference to this issue (#16)

### CODE REVIEW

**Comment by mayank8318** [comment]
@ArihantJain456 Good job.
Can you find a better splash screen than the current one?

File: app/src/main/java/com/iitp/mayank/celesta2k17/activities/SplashActivity.java [hunk 1]
```diff
@@ -13,31 +13,47 @@
...
+
         handler = new Handler();
         action = new Runnable(){
             @Override
             public void run(){
-                Intent intent = new Intent(SplashActivity.this,MainActivity.class);
-                startActivity(intent);
-                finish();
+                splashGif.stop();
+//                Intent intent = new Intent(SplashActivity.this,MainActivity.class);
```
Comment (Muks14x) [inline app/src/main/java/com/iitp/mayank/celesta2k17/activities/SplashActivity.java:48]: Remember to uncomment this out. Looks neat. Try to find a better gif for the splashscreen if you can.

Comment (arihant-001) [reply app/src/main/java/com/iitp/mayank/celesta2k17/activities/SplashActivity.java:48]: oh sorry, I have done this for screenshot .

**Comment by arihant-001** [comment]
there must be a particular logo of your annual fest ? @mayank8318 
we can make something out of it.

**Comment by mayank8318** [comment]
@ArihantJain456  Great idea!!
Logo is what we are using for the icon. You can use it from there.

**Comment by Muks14x** [comment]
@ArihantJain456 Any updates on this?

**Comment by arihant-001** [comment]
do u need gif or only a static image for ur splashscreen @Muks14x

**Comment by Muks14x** [comment]
@ArihantJain456 If you're using the app icon, static would do, along with a circular loader beneath it, or try to get a good looking splash screen gif from the internet to replace the current one.

**Comment by arihant-001** [comment]
I have found these two images 
1. https://i.pinimg.com/originals/b3/8e/04/b38e04064abc6383361e46252373b5e9.gif
2. https://i.pinimg.com/originals/1b/74/ed/1b74ed53670b66f50f4a0a2d4c59f7f5.gif
give ur views...

**Comment by Muks14x** [comment]
@ArihantJain456 The second one looks nice. Can you put it in?

**Comment by arihant-001** [comment]
device-2017-10-02-224035
see...

**Comment by Muks14x** [comment]
@ArihantJain456 Can you commit and push to your branch?

**Pull request closed without merging**